.github/workflows/linting.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       run: |
         cd ${{ github.workspace }}
         pip install ruff
+        ruff format --diff ./
```

Comment:
I guess yo ucan remove ``ruff check`` if you've already done a ``ruff format``, right?

Reply:
`ruff format` doesn't raise an error if it makes any changes, so the subsequent `ruff check` will do that.